src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
```

Comment:
this font is already installed on the body, so it will automatically be applied to all elements, so there is no need to prescribe it to individual elements